go/pkg/log/server/property_test.go

```diff
@@ -280,10 +280,11 @@ func (suite *LogServerTestSuite) modelPurgeLogs(ctx context.Context, t *rapid.T)
 
 func (suite *LogServerTestSuite) modelGarbageCollection(ctx context.Context, t *rapid.T) {
 	for id := range suite.model.CollectionData {
-		exist, err := suite.sysDb.CheckCollection(ctx, id.String())
+		exists, err := suite.sysDb.CheckCollections(ctx, []string{id.String()})
```

Comment:
I'd leave a comment here that we're using a plural API and selecting zero'th element.  It took me a minute to see why this isn't a truncation.